docs/conf.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+#
+# Yolapy documentation build configuration file, created by
+# sphinx-quickstart on Thu Aug 27 12:47:53 2015.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import shlex
+
+from recommonmark.parser import CommonMarkParser
```

Comment:
Removed markdown doc stuff in https://github.com/yola/yolapy/commit/19dbddb74cbe80a280d41871de90dbd0f3fe5e04